frontend/packages/operator-lifecycle-manager-v1/src/fbc/catalog-item.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { CatalogItem } from '@console/dynamic-plugin-sdk/src';
+import { SyncMarkdownView } from '@console/internal/components/markdown-view';
+import { Timestamp } from '@console/internal/components/utils';
+import { CapabilityLevel } from '@console/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details';
+import { validSubscriptionReducer } from '@console/operator-lifecycle-manager/src/components/operator-hub/operator-hub-utils';
+import { ExternalLink } from '@console/shared/src/components/links/ExternalLink';
+import { PlainList } from '@console/shared/src/components/lists/PlainList';
+import { ExtensionCatalogItem } from './types';
+
+type NormalizeExtensionCatalogItem = (item: ExtensionCatalogItem) => CatalogItem;
+export const normalizeExtensionCatalogItem: NormalizeExtensionCatalogItem = (pkg) => {
+  const {
+    catalog,
+    categories,
+    capabilities,
+    description,
+    displayName,
+    icon,
+    infrastructureFeatures,
+    keywords,
+    longDescription,
+    name,
+    provider,
+    repository,
+    support,
+    image,
+    source,
+    validSubscription,
+    createdAt,
+  } = pkg;
+  const [validSubscriptions, validSubscriptionFilters] = validSubscriptionReducer(
+    validSubscription,
+  );
+  return {
+    attributes: {
+      keywords,
+      source,
+      provider,
+      infrastructureFeatures,
+      capabilities,
+      validSubscription: validSubscriptionFilters,
+    },
+    creationTimestamp: createdAt,
+    cta: {
+      label: 'Install',
+      href: `/ecosystem/catalog/install/${catalog}/${name}`,
+    },
+    description: description || longDescription,
+    details: {
+      properties: [
+        {
+          label: 'Capability level',
+          value: <CapabilityLevel capability={capabilities} />,
+        },
+        { label: 'Source', value: source || '-' },
+        { label: 'Provider', value: provider || '-' },
+        {
+          label: 'Infrastructure features',
+          value: infrastructureFeatures?.length ? (
+            <PlainList items={infrastructureFeatures} />
+          ) : (
+            '-'
+          ),
+        },
+        {
+          label: 'Valid subscriptions',
+          value: validSubscriptions?.length ? <PlainList items={validSubscriptions} /> : '-',
+        },
+        {
+          label: 'Repository',
+          value: repository ? <ExternalLink href={repository} /> : '-',
```

Comment:
Shall we add a `text` attribute here? 
```
<ExternalLink href={repository} text={repository}
```

Reply:
Currently it is only a external link icon
![Screenshot 2025-03-10 at 3 57 45 PM](https://github.com/user-attachments/assets/35bd8ca8-c98d-48c9-9f76-49b1b47484bb)